java-logging-httpcomponents/src/main/java/uk/gov/hmcts/reform/logging/httpcomponents/OutboundRequestLoggingInterceptor.java

```diff
@@ -26,7 +26,7 @@ public OutboundRequestLoggingInterceptor() {
         this(systemUTC());
     }
 
-    OutboundRequestLoggingInterceptor(Clock clock) {
+    /* default */ OutboundRequestLoggingInterceptor(Clock clock) {
```

Comment:
what's this for?

Reply:
new pmd checks. all can be reviewed in a single commit ;)